components/Button/Button.jsx

```diff
@@ -55,9 +57,17 @@ const StyledButton = styled.button`
   justify-content: center;
   border-radius: 4px;
 
-  ${props =>
+  ${({ iconOnly }) =>
```

Comment:
Question. Why not pass this props to ButtonIcon and then make a ternary to define the margin value? Right know it's just overwriting the old margin, isn't?  What do you think?

Reply:
When I set iconOnly prop to ButtonIcon an error occurs and some components aren't rendered. I aggree that ButtonIcon shoud be reponsible by its styles, so to solve this I set a class to control this according button receives children.